js/ui/text_box/ui.text_editor.base.js

```diff
@@ -833,9 +835,19 @@ const TextEditorBase = Editor.inherit({
         return this._input();
     },
 
+    _checkKeyPressSubscriptions: function() {
```

Comment:
This isn't the first deprecated option. We have a common mechanism for warning about them. 
Use [_setDeprecatedOptions](https://github.com/DevExpress/DevExtreme/blob/20_1/js/ui/button_group.js#L221) instead of this.